docs/griptape-cloud/data-sources/index.md

```diff
@@ -0,0 +1,21 @@
+# Data Sources
+
+Use [Data Sources](https://cloud.griptape.ai/data-sources) to connect and ingest your own data for use with your AI code.
+
+## Create a Data Source
+
+You can [Create a Data Source](https://cloud.griptape.ai/data-sources/create) by specifying the required configuration for your chosen data source.
+
+If you do not see a data source coniguration you'd wish to use, you can submit a request via Discord or `hello@griptape.ai`.
+
+## Refresh a Data Source
+
+By default your Data Source will not refresh automatically. You can enable automatic refresh and specify a CRON expression when creating a Data Source.
+
+For example, if you wish your Data Source to refresh every day at midnight you can use the following expression: `0 0 * * *`.
+
+If you wish to manually refresh a data source you can do so either via the `Refresh` button on the UI or by API using the `Data Source ID` on the `Config` tab. You will need a [Griptape Cloud API Key](https://cloud.griptape.ai/configuration/api-keys) to make API calls.
+
+```shell
+curl -H "Authorization: Bearer ${GT_CLOUD_API_KEY}" --json '{}' https://cloud.griptape.ai/api/data-connectors/${data_source_id}/data-jobs
```

Comment:
We might want to call out that `${data_source_id}` needs to be replaced in some way.

Reply:
Does this help?

The following shell commands will create a new data refresh job. You will need to specify your API key and data source id.
```shell
export GT_CLOUD_API_KEY=<your API key here>
export DATA_SOURCE_ID=<your data source id here>
curl -H "Authorization: Bearer ${GT_CLOUD_API_KEY}" --json '{}' https://cloud.griptape.ai/api/data-connectors/${DATA_SOURCE_ID}/data-jobs
```